tests/EvmAcceptanceTests/test/precompiles/EvmPrecompiles.js

```diff
@@ -33,4 +33,20 @@ describe("Precompile tests with web3.js", function () {
     const readValue = await contract.methods.idStored().call({gasLimit: 50000});
     expect(readValue).to.be.eq(hash);
   });
+
+  it("When SHA2-256 function is used it should return correct hash", async function () {
```

Comment:
Please fix the description based on the convention we use. we read `it` blocks from the beginning of it. so `it("should return correct hash when SHA2-256 function is used")` is read like this: it should return correct hash when SHA2-256 function is used. that's why the called it `it` :D

Reply:
Sure sure, fixing.